Makefile

```diff
@@ -277,11 +277,16 @@ docker: _print_vars docker-build docker-push
 
 ### docker-build: Build the controller image
 docker-build:
-	docker build . -t ${IMG} -f build/Dockerfile
+	docker build . -t ${DWO_IMG} -f build/Dockerfile
 
 ### docker-push: Push the controller image
 docker-push:
-	docker push ${IMG}
+ifneq ($(INITIATOR),CI)
+ifeq ($(DWO_IMG),quay.io/devfile/devworkspace-controller:next)
+	@echo -n "Are you sure we want to push $(DWO_IMG)? [y/N] " && read ans && [ $${ans:-N} = y ]
```

Comment:
It's strange to me that this occurred in the first place -- I've never been able to push to quay.io/devfile/devworkspace-controller directly (AFAIK). Pushing to this repo should be restricted so that only CI can do it.

Reply:
> I've never been able to push to quay.io/devfile/devworkspace-controller directly (AFAIK).

Haha.
Controller team(You, Josh and I) are admin for that repository and quay.io does not provide an ability to maintain repo but not be able to push there.

I might revoke your permissions there and then you would need to ask me do the stuff, but at this point, CI is set up and I'm not sure which maintaining is expected for that repo.

But in any case this PR would help me to prevent occasional push, or I need separate accounts for administrations and personal testing stuff.

Reply:
:+1: makes sense. I also just realized that I'm never logged in to quay.io, so that would explain why I can't just push there :smile: